src/inputs/internal/ComboBoxInput.tsx

```diff
@@ -57,9 +68,13 @@ export function ComboBoxInput<O, V extends Value>(props: ComboBoxInputProps<O, V
     resetField,
     isTree,
     listBoxRef,
+    inputRef,
+    inputWrapRef,
+    multiline = false,
     ...otherProps
   } = props;
 
+  const { wrap = multiline } = usePresentationContext();
```

Comment:
Kinda curious, seems like we have both `wrap` & `multiline` now...will they always be aligned, i.e. if `wrap=true` then `multiline=true`, and if `wrap=false` then `multiline=false`?

Wondering if like `multiline=false` happens on 73, then they had `wrap=true` set in PC, we'll end up with `wrap=true` but `multiline=false` ... is that okay/a supported configuration?

Reply:
Good question. I glazed over this a bit when doing it so will try to make this a bit simpler / easier to read.

Reply:
Alrighty, just went back over it and added a comment above a new variable `allowWrap`. This variable is going to be true if either `ComboboxInputProps.multiline` is **true**, or `PresentationContext.wrap` is **true**. i.e.
```
const allowWrap = wrap || multiline;
```
For now we'll always expect that if either is set, then we allow it to wrap. Maybe we'll eventually want the local `multiline` prop to override the presentation provider property if it is explicitly set. But I think for now this is fine.